spaces/S000065/properties/P000064.md

```diff
@@ -0,0 +1,12 @@
+---
+space: S000065
+property: P000064
+value: false
+refs:
+- doi: 10.1007/978-1-4612-6290-9_6
+  name: Counterexamples in Topology
+- wikipedia: Baire_space
+  name: Baire space on Wikipedia
+---
+
+Every point has a neighborhood homeomorphic to $[0,1]$, which is locally compact Hausdorff, hence Baire.  And a space that is "locally Baire" is Baire.
```

Comment:
Do we need Hausdorff or regular? See https://topology.pi-base.org/theorems/T000136

Reply:
Adding "locally Baire" as sufficient for Baire to https://topology.pi-base.org/theorems/T000136 may be handy.

Reply:
Actually, can't this be removed with T172?

Reply:
Ohh. it isn't locally Euclidean, it's only homeomorphic to $[0,\infty)$. Annoying.

Reply:
Yeah, initially I thought the same thing, but it's not locally Euclidean.  That's why "line with two origins" will be a good example.